conans/test/utils/runner.py

```diff
@@ -5,7 +5,7 @@ class TestRunner(object):
     def __init__(self, output):
         self._output = output
 
-    def __call__(self, command, cwd=None):
+    def __call__(self, command, output=None, cwd=None):
```

Comment:
Maybe the test runner can be now removed and use also the ConanRunner? I'd say they share a lot. Moreover, it would help to further test the non os.system version of ConanRunner in more cases.